### DISCUSSION TITLE
Phpstorm AlpineJS no variables hightlight

### DISCUSSION THREAD

**Discussion by MaxAlekseyev** [izhangzhihao/intellij-rainbow-brackets#2786] [General]
PHP and pure js syntax highlight exists, but no AlpineJS
- PhpStorm 2024.2.2

![image](https://github.com/user-attachments/assets/4de7d9cd-f32e-4df9-9d2d-f042dca3edfd)

HTML rainbow indents not working also, tried to find by myself but no success

![image](https://github.com/user-attachments/assets/31c5b68a-0a89-4397-8b28-1fa9ef28792d)

**Comment by izhangzhihao** [comment]
What is AlpineJS?

How to reproduce your issue?

**Reply by MaxAlekseyev** [reply]
That's this file https://drive.google.com/file/d/1-0pXZRhFPZUWrzleWPmhUiJ_eaX92sBJ/view?usp=sharing

**Comment by izhangzhihao** [comment]
Can not reproduce, please modify your settings follow: https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions/2595
